Cocoapods/Networking/APIs/Authors/CCPAuthorsAPIManager.h

```diff
@@ -0,0 +1,20 @@
+//
+//  CCPAuthorsAPIManager.h
+//  Cocoapods
+//
+//  Created by Gabriel Massana on 7/2/16.
+//  Copyright © 2016 Gabriel Massana. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <CoreNetworking/CoreNetworking.h>
+
+#import "CCPAbstractAPIManager.h"
+
+@interface CCPAuthorsAPIManager : CCPAbstractAPIManager
+
++ (void)retrieveAuthorsWithString:(NSString *)string
```

Comment:
Not sure about "string", is it being used as a search term or the full author's name? - maybe "searchTerm", "Author" or "Name" for the parameter and "retrieveAuthorsWithName" or "retrieveAuthorsContainingSearchTerm"?